crypto/crypto/src/hash.rs

```diff
@@ -41,8 +41,8 @@
 //! ```
 //! use libra_crypto::hash::CryptoHash;
 //! use libra_crypto_derive::{CryptoHasher, LCSCryptoHash};
-//! use serde::Serialize;
-//! #[derive(Serialize, CryptoHasher, LCSCryptoHash)]
+//! use serde::{Serialize, Deserialize};
```

Comment:
maybe order alphabetically? otherwise it is rather unclear how we are ordering it

Reply:
I see